modules/schematics/schematics-core/utility/parse-name.ts

```diff
@@ -5,7 +5,7 @@ export interface Location {
   path: Path;
 }
 
-export function parseName(path: string, name: string): Location {
+export function parseName(path: string, name: string = ''): Location {
```

Comment:
I think we would want to scope this change specific to the store shematics.
To do this, we should default to an empty string at https://github.com/ngrx/platform/blob/3d0642867d6f4443a3fcfce194bc1112546ededd/modules/schematics/src/store/index.ts#L134

Reply:
I will change this.

Reply:
@timdeschryver i changed it now